Lib/http/cookies.py

```diff
@@ -598,15 +598,19 @@ def __parse_string(self, str, patt=_CookiePattern):
 
 
 class SimpleCookie(BaseCookie):
-    """
-    SimpleCookie supports strings as cookie values.  When setting
+    """SimpleCookie supports strings as cookie values.  When setting
     the value using the dictionary assignment notation, SimpleCookie
     calls the builtin str() to convert the value to a string.  Values
     received from HTTP are kept as strings.
     """
     def value_decode(self, val):
+        """Return an unquoted string, from the cookie header in a reversed fasion of value_encode.
```

Comment:
There is a typo:

```suggestion
        """Return an unquoted string, from the cookie header in a reversed fashion of value_encode.
```

And this line width is larger than 80, we should break it.